internal/impl/gcp/changestreams/dialect.go

```diff
@@ -1,3 +1,16 @@
+// Copyright 2025 Redpanda Data, Inc.
```

Comment:
Can we clarify with @davidfoleyrp the right approach here? Generally I haven't seen us add our license on top of an existing one.

Reply:
Most likely it would be almost completely rewritten in the process.